services/management/node.go

```diff
@@ -63,6 +63,17 @@ func (s *NodeService) Register(ctx context.Context, req *managementpb.RegisterNo
 			return err
 		}
 
+		if node, err := models.CheckUniqueNodeInstanceRegion(tx.Querier, req.Address, req.Region); err != nil {
```

Comment:
`CheckUniqueNodeInstanceRegion` checks that region is not empty string. It can be empty string in that call – Node _can_ be registered without region.

Reply:
This function is not doing anything if region is empty.
https://github.com/percona/pmm-managed/blob/PMM-4919_add_force_not_working/models/node_helpers.go#L68

Reply:
Except that this code wasn't pushed. That's the code that was reviewed: https://github.com/percona/pmm-managed/blob/4f89ef8da58352b0a8670669fea1ba1807ecef26/models/node_helpers.go#L63-L69

Reply:
Yes, you are right. Sorry.